src/radix_connect/wallet_interaction/dapp_wallet_interaction/dapp_to_wallet/interaction_items/transaction/unvalidated_transaction_manifest.rs

```diff
@@ -0,0 +1,98 @@
+use crate::prelude::*;
+
+#[derive(
+    Clone, PartialEq, Eq, Serialize, Deserialize, Debug, uniffi::Record,
+)]
+pub struct UnvalidatedTransactionManifest {
+    #[serde(rename = "transactionManifest")]
+    pub transaction_manifest_string: String,
+
+    pub blobs: Blobs,
+}
+
+impl UnvalidatedTransactionManifest {
+    pub fn new(
+        transaction_manifest_string: impl AsRef<str>,
+        blobs: impl Into<Blobs>,
+    ) -> Self {
+        Self {
+            transaction_manifest_string: transaction_manifest_string
+                .as_ref()
+                .to_owned(),
+            blobs: blobs.into(),
+        }
+    }
+}
+
+impl From<TransactionManifest> for UnvalidatedTransactionManifest {
+    fn from(transaction_manifest: TransactionManifest) -> Self {
+        Self {
+            transaction_manifest_string: transaction_manifest
+                .instructions_string(),
+            blobs: transaction_manifest.blobs().clone(),
+        }
+    }
+}
+
+impl UnvalidatedTransactionManifest {
```

Comment:
Change to:

```rust
impl TryFrom<UnvalidatedTransactionManifest> for TransactionManifest
```

Reply:
hmm... yeah I dont feel to strongly about it, but that is a pattern Ive used in a lot of other places, see:

```
~/Developer/Radix/sargon/src/wrapped_radix_engine_toolkit/low_level/transaction_manifest/transaction_manifest.rs:
   70  }
   71  
   72: impl TryFrom<(ScryptoTransactionManifest, NetworkID)> for TransactionManifest {
   73      type Error = CommonError;
   74      fn try_from(

~/Developer/Radix/sargon/src/wrapped_radix_engine_toolkit/low_level/transaction_manifest/instructions/instructions.rs:
   39  }
   40  
   41: impl TryFrom<(&[ScryptoInstruction], NetworkID)> for Instructions {
   42      type Error = CommonError;
```